Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -133,18 +134,25 @@ Most of the steps for this phase have been automated using the `finish_release`
 
 The hotfix is cut from the latest stable/hotfix release tag (e.g. `4.1.1` from `4.1.0`, or `4.1.2` from `4.1.1`).
 
-**⚠️ NOTE:** Only run the automated process if the previous release has been finished. In order to run the CRP, the GitHub release need to exist, otherwise it will include previous, released tickets.
+**⚠️ NOTE:** Only run the automated process if the previous release has been finished. In order to run the CRP, the **GitHub Release** need to exist (and not just the git tag), otherwise it will include previous, released tickets.
```

Comment:
```suggestion
**⚠️ NOTE:** Only run the automated process if the previous release has been finished. In order to run the CRP, the **GitHub Release** needs to exist (and not just the git tag), otherwise it will include previous, released tickets.
```